### PR TITLE
DM-39813-main: Add 25.0.0 release notes

### PR DESCRIPTION
**Pull request by mwittgen**

### CODE REVIEW

**Comment by mwittgen** [comment]
Build: https://pipelines.lsst.io/v/tickets-DM-39813-main/index.html

File: releases/v25_0_0.rst [hunk 1]
```diff
@@ -1,17 +1,302 @@
...
+
+- :ref:`release-v25-0-0-analysis-tools`
+- :ref:`release-v25-0-0-gbdes`
+- :ref:`release-v25-0-0-parsl-plugin`
+- :ref:`release-v25-0-0-hips`
+
+.. _release-v25-0-0-analysis-tools:
+
+Analysis Tools
+--------------
```
Comment (jonathansick) [inline releases/v25_0_0.rst:59]: Remember to add a blank line after section underlines (check all sections).

File: releases/v25_0_0.rst [hunk 2]
```diff
@@ -1,17 +1,302 @@
...
+
+
+This release is based on the ``w_2022_48`` weekly build with 18 tickets backported.
+
+The notes below highlight significant technical changes to the Science Pipelines codebase in this release.
+For a complete list of changes made, see :doc:`tickets/v25_0_0`.
+
+The `Characterization Metric Report (DMTR-302) <https://dmtr-392.lsst.io/>`_ describes the scientific performance of this release in terms of scientific performance metrics.
+
+If you have questions or comments about this release, visit our `community forum <https://community.lsst.org>`_ for advice.
```
Comment (jonathansick) [inline releases/v25_0_0.rst:31]:
```suggestion
If you have questions or comments about this release, visit our `Community forum <https://community.lsst.org>`_ for advice.
```

File: releases/v25_0_0.rst [hunk 3]
```diff
@@ -1,17 +1,302 @@
...
+Hierarchical Progressive Surveys (HiPS) trees
+---------------------------------------------
+
+`~lsst.pipe.tasks.hips.GenerateHipsTask` was added to the pipelines.
+This Task generates Hierarchical Progressive Surveys (HiPS) trees from high-resolution, healpix-projected coadds. HiPS images produced by this Task are viewable in the Portal for `Data Preview 0.2 <https://dp0-2.lsst.io/>`_.
+
+
+.. _release-v25-0-0-interface:
+
+Significant Interface Changes
```
Comment (jonathansick) [inline releases/v25_0_0.rst:88]: Remember we use sentence case for section headings per our Developer Guide and https://developers.google.com/style/highlights (look in all sections)

File: releases/v25_0_0.rst [hunk 4]
```diff
@@ -1,17 +1,302 @@
...
+- :ref:`release-v25-0-0-deprecate-ip-diffim`
+- :ref:`release-v25-0-0-deprecate-afw-table`
+- :ref:`release-v25-0-0-deprecate-daf-butler`
+- :ref:`release-v25-0-0-deprecate-configurations`
+
+.. _release-v25-0-0-deprecate-obs-base:
+
+Deprecate Method getInstrument
+------------------------------
+``obs.base.util.getInstrument`` will be replaced by ``lsst.pipe.base.Instrument.from_string``
```
Comment (jonathansick) [inline releases/v25_0_0.rst:148]: One-sentence per line is preferred if possible (check all content).

File: releases/v25_0_0.rst [hunk 5]
```diff
@@ -1,17 +1,302 @@
...
+``Butler.datasetExistsDirect`` will be replaced by ``Butler.stored``.
+``Butler.datasetExists`` will be replaced by ``Butler.exists``.
+The ``Butler.datastore`` property will be deprecated.
+The ``DimensionPacker`` class will be deprecated in v26.0.0 in favor of configurable dimension packers.
+Before mentioned will be removed after the release of v27.0.0.
+
+.. _release-v25-0-0-deprecate-skymap-packers:
+
+Deprecate SkyMapDimensionPacker
+--------------------------------------------
```
Comment (jonathansick) [inline releases/v25_0_0.rst:189]: Check that section heading underlines are the same length as the section (check all headings).

**Review by jonathansick — CHANGES_REQUESTED** [review]

**Comment by mwittgen** [comment]
Fixes were made in DM-39813 and cherry-picked to this branch

**Review by jonathansick — APPROVED** [review]
Looks great.

**Pull request merged**